static/bidder-info/taboola.yaml

```diff
@@ -6,9 +6,11 @@ capabilities:
   app:
     mediaTypes:
       - banner
+      - native
   site:
     mediaTypes:
       - banner
+      - native
```

Comment:
Please update adapter documentation accordingly: https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/taboola.md

Reply:
will do once we implement the changes in the Prebid.js adapter

Reply:
Need to confirm it with the team.